serverlessworkflow/modules/ROOT/pages/supporting-services/jobs-service.adoc

```diff
@@ -0,0 +1,107 @@
+Additionally, it provides an initialization procedure that integrates link:https://flywaydb.org[Flyway] for the database initialization. It automatically controls the database schema, in this way all tables are created by the service.
+
+In case you need to externally control the database schema, you can check and apply the DDL scripts for the Job Service in the same way as described in
+xref:persistence/postgresql-flyway-migration.adoc#manually-executing-scripts[Manually executing scripts] guide.
+
+You need to set the proper configuration parameters when starting the Job Service.
+The xref:use-cases/timeout-showcase-example.adoc[timeout showcase example] shows how to run PostgreSQL as a Kubernetes deployment, but you can run it the way it fits in your environment, the important part is to set all the configuration parameters points to your running instance of PostgreSQL.
+
+==== Ephemeral
+Alternatively, there is an in-memory database support that does not require any external database configuration, it can be used for testing and development purposes, but it is not recommended for production, since all jobs are lost upon a service restart or failure.
+
+[NOTE]
+====
+The Job Service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and Ephemeral (InMemory) options that can be switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `ephemeral` option. For more information about the Job Service, the container image can be found link:{jobs_service_image_usage_url}[here].
+====
+
+== Job service leader election
+
+Currently, the Job Service works in a single instance manner where there should be just one active instance of the service.
+
+To avoid issues when deploying the service in the cloud, where it is common to eventually have more than one instance deployed, the Job Service supports a leader instance election process. Only the instance that become the leader activates the external communication to receive and schedule jobs.
+
+All instances who are not leaders, stay inactive in a kind of wait state and try to become the leader continuously.
+
+When a new instance of the service is started it is not set as a leader at startup time but instead, it starts the process to become one.
+
+When an instance that is the leader for any issue stays unresponsive or it is shut down, one of the other running instances becomes the leader.
+
+.Job Service leader election
+image::core/job-service-leader.png[]
+
+[NOTE]
+====
+This leader election mechanism uses the underlying persistence backend, which currently is only supported in the PostgreSQL implementation.
+====
+
+There is no need for any configuration to support this feature, the only requirement is to have the supported database with the data schema up-to-date as described in the <<job-service-postgresql>> section.
+
+In case the underlying persistence does not support this feature, you should guarantee
```

Comment:
```suggestion
In case the underlying persistence does not support this feature, you must guarantee that just one single instance of the Job Service is running at the same time.
```